dapps/marketplace/src/constants/SystemInfo.js

```diff
@@ -0,0 +1 @@
+export const rewardsOnMobileEnabled = true
```

Comment:
Do we need this as a separate file? Seems it's only used once

Reply:
I have separated it into a file because it is used in 2 places: 
- `dapps/marketplace/src/pages/growth/Welcome.js`
- `dapps/marketplace/src/pages/user/ProfileWizard.js`

Reply:
Ah yea ProfileWizard was too big to show in the diff 🙂